CHANGELOG.md

```diff
@@ -6,6 +6,12 @@
 
 Users with access to the new Mac hardware should review [important information provided here](https://cromwell.readthedocs.io/en/stable/Releases).
 
+### Experimental SQLite support
+
+In addition to unsupported and experimental HSQLDB file support, unsupported and experimental SQLite file support has
```

Comment:
"Unsupported support" sounds weird, so I'd leave only "experimental" part here or somehow rephrase this.